training/datasets.py

```diff
@@ -324,8 +324,8 @@ def preprocess_sample(self, seq_idx, first_idx, second_idx):
         ref_w = (ref_annot['xmax'] - ref_annot['xmin'])/2
         ref_h = (ref_annot['ymax'] - ref_annot['ymin'])/2
         ref_ctx_size = self.ref_context_size(ref_h, ref_w)
-        ref_cx = (ref_annot['xmax']+ref_annot['xmin'])/2
-        ref_cy = (ref_annot['ymax']+ref_annot['ymin'])/2
```

Comment:
I don't quite get what you did here. Weren't it supposed to remove the /2 factor from the refs?